src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.component.ts

```diff
@@ -617,4 +659,6 @@ export class CodeEditorAceComponent implements AfterViewInit, OnChanges, OnDestr
     updateTabSize(event: number) {
         this.tabSize = event;
     }
+
+    protected readonly Feedback = Feedback;
```

Comment:
What is this for? Why is it at the very end of the class instead of near the top with other attributes?

Reply:
It allows the HTML code (the template) to access the `Feedback` class. I've moved it to the top and added a comment (also in similar places).